.github/workflows/test.yaml

```diff
@@ -44,3 +44,11 @@ jobs:
       - uses: coursier/cache-action@v6
       - name: Compile
         run: sbt vyxalJS/compile
+      - name: Link
+        run: sbt vyxalJS/fastLinkJS
```

Comment:
Why not `fullLinkJS`?

Reply:
`fastLinkJS` is faster, and these aren't production builds; they're mostly for checking that the code does what you want without compiling it yourself